### ISSUE TITLE
[Discourse] Cannot use plugins Oauth2 and OpenId

### ISSUE THREAD

**Issue by jbtruffault** [bitnami/vms#505]
### Platform

Installers

### bndiagnostic ID [know more about bndiagnostic ID](https://docs.bitnami.com/general/how-to/understand-bndiagnostic/)

docker.io/bitnami/discourse:2.8.9

### bndiagnostic output

I couldn't get ID from the tool, here is the output:

Warning: The path does not seem to contain a valid Bitnami installation! Make
sure you have proper access rights. 
Press [Enter] to continue:cd /opt/bitnami
----------------------------------------------------------------------------

Here is my /opt/bitnami folder:

root@cb234b9df628:/opt/bitnami# ls
bndiagnostic  bndiagnostic-tool  brotli  common  **discourse**  licenses  node  postgresql  python  ruby  scripts

I'm using the image: docker.io/bitnami/discourse:2.8.9

### bndiagnostic was not useful. Could you please tell us why?

Couldn't find the bitnami folder

### Describe your issue as much as you can

I'm trying to configure SSO to connect to my discourse instance

I first installed this plugin: https://github.com/discourse/discourse-openid-connect by following the steps described by discourse editor for plugin installation. 

I got this error when I tried to connect with OpenId. It appears on the following URL: https://<discourse-instance.com>/auth/oidc.

![image](https://user-images.githubusercontent.com/12747168/203149426-ce5b7a24-ea6d-43cc-80b8-9628ac11a8d4.png)

Some details of the POST request (error 500)
![image](https://user-images.githubusercontent.com/12747168/203149769-e150a7ab-6c9e-4b78-90ca-94b1cc9b5b04.png)

The thing is: I encountered this same error by installing the plugin https://github.com/discourse/discourse-oauth2-basic and trying to connect through Oauth2.

Some details from the log:

```
Started POST "/auth/oauth2_basic" for 80.215.43.80 at 2022-11-22 08:12:40 +0000
(oauth2_basic) Setup endpoint detected, running now.
(oauth2_basic) Request phase initiated.
NameError (uninitialized constant FinalDestination::FaradayAdapter)
lib/middleware/omniauth_bypass_middleware.rb:50:in `call'
lib/content_security_policy/middleware.rb:12:in `call'
lib/middleware/anonymous_cache.rb:361:in `call'
config/initializers/100-quiet_logger.rb:23:in `call'
config/initializers/100-silence_logger.rb:31:in `call'
lib/middleware/enforce_hostname.rb:23:in `call'
lib/middleware/request_tracker.rb:198:in `call'
Started GET "/favicon.ico" for 80.215.43.80 at 2022-11-22 08:12:40 +0000
ActionController::RoutingError (No route matches [GET] "/favicon.ico")
config/initializers/100-quiet_logger.rb:23:in `call'
config/initializers/100-silence_logger.rb:31:in `call'
lib/middleware/enforce_hostname.rb:23:in `call'
lib/middleware/request_tracker.rb:198:in `call'
  Rendered exceptions/not_found.html.erb within layouts/no_ember (Duration: 87.0ms | Allocations: 11001)
  Rendered layout layouts/no_ember.html.erb (Duration: 371.0ms | Allocations: 70119)
  Rendered html template (Duration: 0.0ms | Allocations: 2)
```

This make me thinking that it may comes from the Discourse Core. 

This thread also describes the error, trying to involve discourse developpers: https://meta.discourse.org/t/error-of-discourse-openid-connect/245465/6 But they may not answer because installation relies on bitnami docker. And this is my point: do you think this error can be specific bitnami image ? Would you have any idea to debug ?

Best regards

**Comment by jbtruffault** [comment]
This has been solved with docker version 2.8.11 published just after 2.8.9

Thanks !

**Comment by jbtruffault** [comment]
``
